docs/api/utils.rst

```diff
@@ -0,0 +1,604 @@
+:meth:`EMPTY_IRECT`                  return the (standard) empty / invalid rectangle
+:meth:`EMPTY_QUAD`                   return the (standard) empty / invalid quad
+:meth:`EMPTY_RECT`                   return the (standard) empty / invalid rectangle
+:meth:`INFINITE_IRECT`               return the (only existing) infinite rectangle
+:meth:`INFINITE_QUAD`                return the (only existing) infinite quad
+:meth:`INFINITE_RECT`                return the (only existing) infinite rectangle
+:meth:`GetPdfNow`                    return the current timestamp in PDF format
+:meth:`GetPdfString`                 return PDF-compatible string
+:meth:`GetTextLength`                return string length for a given font & :data:`fontsize`
+:meth:`GlyphName2Unicode`            return unicode from a glyph name
+:meth:`GetImageProfile`              return a dictionary of basic image properties
+:meth:`INFINITE_RECT`                return the (only existing) infinite rectangle
+:meth:`PaperRect`                    return rectangle for a known paper format
+:meth:`PaperSize`                    return width, height for a known paper format
+:meth:`PaperSizes`                   dictionary of pre-defined paper formats
+:meth:`PlanishLine`                  matrix to map a line to the x-axis
+:meth:`RecoverCharQuad`              compute the quad of a char ("rawdict")
+:meth:`RecoverLineQuad`              compute the quad of a subset of line spans
+:meth:`RecoverQuad`                  compute the quad of a span ("dict", "rawdict")
+:meth:`RecoverSpanQuad`              compute the quad of a subset of span characters
+:meth:`Unicode2GlyphName`            return glyph name from a unicode
+:meth:`MakeAnnotDA`                  Passing color, fontname, fontsize into the annot
+:meth:`AddAnnotId`                   Add a unique /NM key to an annotation or widget
+:meth:`AddOcObject`                  Add OC object reference to a dictionary
+:meth:`ColorCount`                   Return count of each color.
+:meth:`BinFromBuffer`                Turn FzBuffer into a byte[]
+:meth:`BufferFromBytes`              Make FzBuffer from a byte[] object
+:meth:`CalcImageMatrix`              Compute image insertion matrix
+:meth:`CompressBuffer`               Compress FzBuffer into a new buffer
+:meth:`GetPdfString`                 Make a PDF-compatible string
+:meth:`ConstructLabel`               Construct a label based on style, prefix and page number
+:meth:`DecodeRawUnicodeEscape`       Decode raw unicode
+:meth:`DoLinks`                      Insert links contained in copied page range into destination PDF
+:meth:`EnsureIdentity`               Store ID in PDF trailer
+:meth:`ExpandFontName`               Make /DA string of annotation
+:meth:`GetId`                        Count numbers and return unique id on one process
+:meth:`GetAllContents`               All /Contents streams concatenated to one bytes object.
```

Comment:
why is there a full stop at the end of this sentence? Either we full stop at the end of all these method descriptions or we don't.